examples/never-type/never_type.bal

```diff
@@ -0,0 +1,44 @@
+import ballerina/io;
+import ballerina/lang.'xml as xmllib;
+
+type Person record {
+    string name;
+    int age;
+};
+
+type Student record {
+    string name;
+    // You can specify an optional field with `never` type, but can not assign a value.
+    never gender?;
+};
+
+// This function never returns. So we have return type of `never`.
+function validateAndPanic(int age) returns never {
+    if (age < 18) {
+        error e = error("Invalid Age");
+        panic e;
+    }
+}
+
+public function main() {
+    // We can call this function, but can not assign to a value.
+    validateAndPanic(20);
+    io:println(typeof validateAndPanic);
+
+    // This is an example of use of `never` type as a parameter in `xml`. 
+    // `xml<never>` describes the `xml` type that has no constituents,
+    // i.e. the empty xml value.
+    xml<never> xmlValue = <xml<never>> xmllib:concat();
```

Comment:
we don't need to import xml module now as it's pre-imported.
You should be able to say `xml<never> xmlValue = <xml<never>> 'xml:concat();`

Reply:
ack